form/without_class.rst

```diff
@@ -123,3 +126,56 @@ but here's a short example::
     When a form is only partially submitted (for example, in an HTTP PATCH
     request), only the constraints from the submitted form fields will be
     evaluated.
+
+Constraints At Class Level
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another possibility is to add the constraints at the class level.
+This can be done by setting the ``constraints`` option in the
+``configureOptions()`` method::
+
+    use Symfony\Component\Form\Extension\Core\Type\TextType;
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\Validator\Constraints\Length;
+    use Symfony\Component\Validator\Constraints\NotBlank;
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('firstName', TextType::class)
+            ->add('lastName', TextType::class)
+        ;
+    }
+
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $constraints = [
+            'firstName' => new Length(['min' => 3]),
+            'lastName' => [
+                new NotBlank(),
+                new Length(['min' => 3]),
+            ],
+        ];
+
+        $resolver->setDefaults([
+            'data_class' => null,
+            'constraints' => $constraints,
+        ]);
+    }
+
+This means you can also do this when using the ``createFormBuilder()`` method
+in your controller::
+
+    $form = $this->createFormBuilder($defaultData, [
+            'constraints' => [
+                'firstName' => new Length(['min' => 3]),
+                'lastName' => [
+                    new NotBlank(),
+                    new Length(['min' => 3]),
+                ],
+            ],
+        ])
+        ->add('firstName', TextType::class)
+        ->add('lastName', TextType::class)
+        ->getForm()
+    ;
```

Comment:
```suggestion
        ->getForm();
```

Reply:
Should we update other examples to follow this code style?

Reply:
Not sure, I just want to make it more compact.

@javiereguiluz WDYT?